command/daemon.go

```diff
@@ -92,8 +100,22 @@ func daemonCommand(cctx *cli.Context) error {
 	if err != nil {
 		return err
 	}
+
+	gsnet := gsnet.NewFromLibp2pHost(h)
+	gs := gsimpl.New(context.Background(), gsnet, cidlink.DefaultLinkSystem())
+	tp := gstransport.NewTransport(h.ID(), gs)
+	dtNet := dtnetwork.NewFromLibp2pHost(h)
+	tmpDir, err := ioutil.TempDir("", "indexer-dt-dir")
```

Comment:
Is this a directory that we may want to reuse across separate runs of the reference-provider daemon?

Reply:
eventually yes, but it will also be dependent on who's running it. My understanding is that in Lotus, we won't actually be running the Daemon cause it will just be part of the markets binary.